### DISCUSSION TITLE
Phoniebox for the whole family with bluetooth support and UPS

### DISCUSSION THREAD

**Discussion by TabForce** [MiczFlor/RPi-Jukebox-RFID#1463] [Ideas]
Hello Phonieboxers,
I've made a Phoniebox with your support and wanted to contribute with a few things I've done to make my Phoniebox an awesome device for the little ones as well as grown ups.

I've installed this UPS: https://wiki.geekworm.com/X750, and added a little script at  /usr/lib/systemd/system-shutdown/, to make the device shut down correctly
I'm using a HDMI Audio converter for clear audio,
USB powered Trust Arys 2.0 USB Speakers (connected to the x750)
GPIO Buttons
Used [this](https://www.raspberrypi.org/forums/viewtopic.php?t=235519) tutorial to make the phoniebox acessible as a a2dp sink (bluetooth-speaker)
All powered with this Powersupply: Capolida 5V 10A 50W 100-240V
I've installed a extensionboard for the x750 and can use the phoniebox all day long without a socket.
The x750 should power down automatically and safely on low battery, but the python-script delivered with the x750 won't work with all the python things installed by phoniebox - but this issue will just be a question of time.
(I've put it in this case: https://www.amazon.de/gp/product/B00ZNG3OQI/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1)

If someone needs help or wants to rebuild my version of the phoniebox, feel free to ask

Thank you!
Ray

**Comment by jbehler** [comment]
Is the Bluetooth speaker option going to be added in the future 3 release?
